src/MediaTypes.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the ApiTestCase package.
+ *
+ * (c) Lakion
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Lakion\ApiTestCase;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+final class MediaTypes
```

Comment:
I think about some kind of extension guesser, that's why i create separate class.